embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/LogRecordExporterTest.kt

```diff
@@ -60,10 +63,7 @@ internal class LogRecordExporterTest {
 
                 sleep(3000)
             }
-            Assert.assertTrue((fakeLogRecordExporter.exportedLogs?.size ?: 0) == 0)
-            Assert.assertTrue(fake.msgQueue.any {
```

Comment:
If there were previous test logic that was inspecting what internal errors were logged, how do we do this now?

Reply:
This can be achieved with changes in #828 as it replaces the logger with an interface. IMO the check isn't super helpful here as it's just asserting that a message was logged. I would like to make it clearer that we are logging internal errors & make it easier to test, but that come in additional refactoring

Reply:
Got it. There should be other side effects that can be observed, eh? I suppose if logging is the only thing that isn't verified, it's not that bad.